akuelles Omega-Build bricht bei 93% ab

Habe auf meinem Pi4 Kodi installiert und möchte nun das pvr.waipu nutzen. Zunächst installierte ich die Piers-Version - bekam dann aber im Kodi beim Aufruf des AddOns die Fehlermeldung, dass das pvr.waipu nicht zur Kodi-Version passt. Da ich anscheinend eine Omega-Version von Kodi (20.5) nutze versuchte ich nach einer Rücksicherung ohne Piers-Installationsversuch die aktuelle Omega-Version zu installieren.

Es kommt beim "make" nach ca. 93 % folgende Fehlermeldung:

In file included from /root/pvr.waipu/src/WaipuData.cpp:22:
/root/pvr.waipu/src/WaipuData.h:93:13: error: ‘PVR_ERROR WaipuData::GetChannelStreamProperties(const kodi::addon::PVRChannel&, std::vector<kodi::addon::PVRStreamProperty>&)’ marked ‘override’, but does not override
   93 |   PVR_ERROR GetChannelStreamProperties(
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~
make[5]: *** [CMakeFiles/pvr.waipu.dir/build.make:160: CMakeFiles/pvr.waipu.dir/src/WaipuData.cpp.o] Fehler 1
make[4]: *** [CMakeFiles/Makefile2:110: CMakeFiles/pvr.waipu.dir/all] Fehler 2
make[3]: *** [Makefile:156: all] Fehler 2
make[2]: *** [CMakeFiles/pvr.waipu.dir/build.make:88: pvr.waipu-prefix/src/pvr.waipu-stamp/pvr.waipu-build] Fehler 2
make[1]: *** [CMakeFiles/Makefile2:132: CMakeFiles/pvr.waipu.dir/all] Fehler 2
make: *** [Makefile:91: all] Fehler 2

Die gesamte Installation siehe:
[error_Build.txt](https://github.com/user-attachments/files/18232006/error_Build.txt)

Hatte dann die Vermutung, dass das mit 
`sudo apt-get install kodi`
installierte Kodi 20.5.0 wohl auch Nexus sein könnte - habe daher auch die Installation nach erneuter Rücksicherung mit Nexus versucht - zeigt gleichen Error.

Vielen Dank für Unterstützung!


**Environment / Umgebung:**
 - Kodi version: 20.5.0
 - OS: Raspberry Pi OS Debian version: 12 (bookworm) 
 - pvr.waipu version [?? ]
 - inputstream.adaptive version [??]

Hallo,
habe heute noch die Kodi-Version 21.1 auf dem Rasperry installiert. Hier bricht die Omega-Installation von pvr.waipu ebenfalls bei "make" mit gleichem Fehler ab.

`[ 75%] Building CXX object CMakeFiles/pvr.waipu.dir/src/categories.cpp.o
[ 87%] Building CXX object CMakeFiles/pvr.waipu.dir/src/WaipuData.cpp.o
In file included from /root/pvr.waipu/src/WaipuData.cpp:22:
/root/pvr.waipu/src/WaipuData.h:93:13: error: ‘PVR_ERROR WaipuData::GetChannelStreamProperties(const kodi::addon::PVRChannel&, std::vector<kodi::addon::PVRStreamProperty>&)’ marked ‘override’, but does not override
   93 |   PVR_ERROR GetChannelStreamProperties(
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~
make[5]: *** [CMakeFiles/pvr.waipu.dir/build.make:160: CMakeFiles/pvr.waipu.dir/src/WaipuData.cpp.o] Fehler 1
make[4]: *** [CMakeFiles/Makefile2:110: CMakeFiles/pvr.waipu.dir/all] Fehler 2
make[3]: *** [Makefile:156: all] Fehler 2
make[2]: *** [CMakeFiles/pvr.waipu.dir/build.make:88: pvr.waipu-prefix/src/pvr.waipu-stamp/pvr.waipu-build] Fehler 2
make[1]: *** [CMakeFiles/Makefile2:132: CMakeFiles/pvr.waipu.dir/all] Fehler 2
make: *** [Makefile:91: all] Fehler 2

Environment / Umgebung:

    Kodi version: 21.1.0
    OS: Raspberry Pi OS Debian version: 12 (bookworm)
`

--- a/README.md
+++ b/README.md
@@ -25,7 +25,7 @@ For any support regarding this plugin, please create a github issue.
 
 ### Linux
 
-1. `git clone --branch master https://github.com/xbmc/xbmc.git`
+1. `git clone --branch Omega https://github.com/xbmc/xbmc.git`
 2. `git clone --branch Omega https://github.com/flubshi/pvr.waipu.git`
 3. `cd pvr.waipu && mkdir build && cd build`
 4. `cmake -DADDONS_TO_BUILD=pvr.waipu -DADDON_SRC_PREFIX=../.. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=../../xbmc/addons -DPACKAGE_ZIP=1 ../../xbmc/cmake/addons`

